code/game/objects/items/religion.dm

```diff
@@ -8,30 +8,35 @@
 	attack_verb = list("forcefully inspired", "violently encouraged", "relentlessly galvanized")
 	lefthand_file = 'icons/mob/inhands/equipment/banners_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/banners_righthand.dmi'
+	var/inspiration_available = TRUE //If this banner can be used to inspire crew
```

Comment:
name/desc/examine/something should indicate to players whether or not this is set